src/directives/google-map.ts

```diff
@@ -191,10 +191,12 @@ export class SebmGoogleMap implements OnChanges,
     });
   }
 
+  get wrapper(): GoogleMapsAPIWrapper { return this._mapsWrapper; }
```

Comment:
@alexweber I don't understand the need for this. You can already get the GoogleMapsAPIWrapper  when you create a custom component and inject it in the constructor.